variables.tf

```diff
@@ -109,3 +109,32 @@ variable "tags" {
   default     = {}
   type        = map(string)
 }
+
+########################################
+# Monitoring
+########################################
+variable "monitor_action_group_id" {
+  default     = ""
+  description = "ID of Azure Monitor Action Group for metric to trigger"
+  type        = string
+}
+
+variable "monitor_metric_alert_criteria" {
+  default     = {}
+  description = "Map of name = criteria objects"
+
+  type = map(object({
```

Comment:
Love this. Should we update the example to include a simple one of these?

Reply:
:+1: I slightly edited the client code and pushed it up onto the readme